content/rs/administering/access-control/user-roles.md

```diff
@@ -5,83 +5,73 @@ weight: $weight
 
-{{< embed-html "account-role-table.html" >}}
+### Configuring Roles & Users
 
-### Assigning Management Roles to a User Role
+In **access control > roles**, you can configure various types of user roles. For example:
 
-To assign a management role to a user role:
+- Management roles that define user access to the Admin UI and cluster API
+- Data access controls with Redis ACLs that define the commands and keys that users can access when connecting to databases
+- The databases that you would like each user to be able to access
 
-1. In **access control** > **roles**:
-    - Edit an existing role - Hover over a role and click ![Edit](/images/rc/icon_edit.png#no-click "Edit").
-    - Create a new role - Click ![Add](/images/rs/icon_add.png#no-click "Add").
-1. Select the management role for the user role.
-1. Click **Save**.
+### Management Roles
 
-### User Roles for Database Connections Only
+Redis Enterprise comes with the following roles for the admin console:
 
-To create a user role for users that cannot connect to the RS web UI and API, assign the **None** management role to the user role.
+{{< embed-html "account-role-table.html" >}}
 
-## Database Access Control
+{{% note %}}
+To create a user role that only has access to the database, assign the user the management role **None**
+{{% /note %}}
 
-To control user access to Redis database commands and keys,
-you must define Redis ACLs that specify the commands that users can run and keys that the commands can apply to.
-Then, in the user role, select the databases that the users can access and Redis ACL that controls user access to those databases.
+### Database Access Controls
 
-A-A cannot assign ACLs on creation
+ACLs allow you to control what level of access each user has in Redis. With ACLs, you can specify which commands specific users can execute and which keys they can access. This allows for much better security practices: you can restrict any given user’s access to the least level of privilege needed.
 
 {{% note %}}
+There are several important differences with Redis Enterprise you should be aware of:
 
-- Redis ACLs can only be configured in the cluster web UI or API.
-    In Redis:
-    - These ACL subcommands are blocked: LOAD, SAVE, SETUSER, DELUSER, GENPASS, LOG
-    - These ACL subcommands are allowed: LIST, USER, GETUSER, CAT, WHOAMI, HELP
-- The MULTI, EXEC, DISCARD commands are always allowed, but ACLs are enforced on MULTI subcommands.
-- External users cannot authenticate with databases.
-- When you run multi-key commands on multi-slot keys, the return value is `failure` but the command runs on the keys that are allowed.
-
+1. The ACL subcommands LOAD, SAVE, SETUSER, DELUSER, GENPASS and LOG are blocked
+2. The MULTI, EXEC, DISCARD commands are always allowed, but ACLs are enforced within a transaction
+3. When you run multi-key commands on multi-slot keys, the return value is `failure` but the command runs on the keys that are allowed.
```

Comment:
Let's add here that you can edit this as configuration per DB to block or allow such cases.
Using rladmin:
rladmin tune db <db:id> mkms <enabled/disabled>
Also important: inherent commands restrictions for multi-keys commands such as in OSS Cluster mode or in Active-Active will apply even when this setting (mkms) is turned ON.

Reply:
@AlonMagrafta - would you be able to take a swing at this. I don't understand it fully enough to do it the right level of justice.

Reply:
Took it :)